### ISSUE TITLE
Open API file after the API generation build task ends

### ISSUE THREAD

**Issue by sentry07** [sentry07/vscode-crestron-splus#11]
Generating an API file should open the file. However the newest versions of SplusCC don't seem to be dropping the splusheader.exe file in the SPlsWork folder so we may have to package the binary with the extension.

Code is working, won't merge until we figure out what to do.

**Comment by fordfraipont** [comment]
Hey, was there ever a resolution to this? I just started using this extension yesterday and every time I attempt to generate the API file I get this error: ""______\Program\SPlsWork\splusheader.exe"' is not recognized as an internal or external command,
operable program or batch file.""

**Comment by sentry07** [comment]
No, I will probably just remove that build feature. Crestron stopped dropping the splusheader.exe file in the SPlsWork folder so we can't use it anymore and it's not something that is readily available without unzipping a password protected .ZIP file.

**Comment by alvaroyurrita** [comment]
I had made updates to this extension and is currently on alpha.  It fixed this issue and added more.  Find the release notes [here](https://github.com/alvaroyurrita/vscode-crestron-splus/releases/tag/v1.0.0-alpha8), if you want to give it try.  I will be going beta and pre release during the coming months based on feedback from users such as yourself. Here's the list of things that got added:

Case and Brace autoformat addition
Location Sensitive AutoCompleter
API auto generation
API and External Library AutoComplete
Keywords and Builtin elements AutoComplete
Custom VSCode Task Creation
Document specific Build
Category Insertion
Hover Over Online Help
Revamp SIMPL+ and API Syntax
Function Signature Helper
